Configurations/50-masm.conf

```diff
@@ -11,9 +11,10 @@ my %targets = (
     "VC-WIN64A-masm" => {
         inherit_from    => [ "VC-WIN64-common", asm("x86_64_asm"),
                              sub { $disabled{shared} ? () : "x86_64_uplink" } ],
-        as              => "ml64",
-        asflags         => "/nologo /c /Cp /Cx /Zi",
+        AS              => "ml64",
+        ASFLAGS         => "/nologo",
         asoutflag       => "/Fo",
+        asflags         => "/c /Cp /Cx /Zi",
```

Comment:
We have moved /Zi to ASFLAGS in 10-main.conf. It would be appropriate to follow the lead...

Reply:
Fixed